builder/build.sh

```diff
@@ -101,6 +117,7 @@ _version_is_greater_than() {
 set_up_environment
 fetch_r_source $R_VERSION
 compile_r $R_VERSION
+package_r $R_VERSION
```

Comment:
I think `clean_r` runs some special install steps for CentOS 6 specifically, so it should come before the packaging step (@jonyoder can confirm).

Reply:
Yes, I would agree. The `clean_r` step is really important for CentOS.

Reply:
✅ Done - changed the order of the steps